metricsbp/config.go

```diff
@@ -63,7 +63,8 @@ type Config struct {
 // with the global tracing hook registry.
 func InitFromConfig(ctx context.Context, cfg Config) io.Closer {
 	M = NewStatsd(ctx, cfg)
-	tracing.RegisterCreateServerSpanHooks(CreateServerSpanHook{Metrics: M})
+	pm := NewPrometheusMetrics(ctx, cfg)
+	tracing.RegisterCreateServerSpanHooks(CreateServerSpanHook{Metrics: M, PrometheusMetrics: pm})
```

Comment:
as a style nit, I feel that the only appropriate case of writing a whole struct in one line is when you only fill zero or one of its fields. anything >1 should be write in one-per-line instead:
```suggestion
	tracing.RegisterCreateServerSpanHooks(CreateServerSpanHook{
		Metrics: M,
		PrometheusMetrics: pm,
	})
```